README.md

```diff
@@ -35,6 +35,7 @@ prebuild-install [options]
   --runtime     -r  runtime     (Node runtime [node or electron] to build or install for, default is node)
   --path        -p  path        (make a prebuild-install here)
   --token       -T  gh-token    (github token for private repos)
+  --tag-prefix [prefix=v]       (github tag prefix)
```

Comment:
This syntax suggests the argument is optional, e.g. like `prebuild --tag-prefix` is valid.

I think `--tag-prefix <prefix> (github tag prefix, default is "v")` would be clearer.